inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/extract/KafkaExtractNode.java

```diff
@@ -75,6 +75,9 @@ public class KafkaExtractNode extends ExtractNode implements Metadata, Serializa
     @JsonProperty("groupId")
     private String groupId;
 
+    @JsonProperty("kafkaScanSpecificOffset")
+    private String kafkaScanSpecificOffset;
```

Comment:
This is a Kafka extract node, so just calling it `scanSpecificOffset` is enough.

Reply:
OK, thanks for your advice